pontoon/base/utils.py

```diff
@@ -677,8 +677,7 @@ def glob_to_regex(glob):
 
 def get_m2m_changes(current_qs, new_qs):
     """
-    Get difference between states of m2m relation.
-    When formfield_items is empty, return all items from object_items as removed.
+    Get difference between states of a m2m relation.
```

Comment:
Can you change `m2m` to `many to many` in this comment? I now remember that it took me a few moments to figure out what "m2m" means.

Reply:
:+1: